src/main/java/me/champeau/gradle/japicmp/ignore/processor/Changer.java

```diff
@@ -0,0 +1,49 @@
+package me.champeau.gradle.japicmp.ignore.processor;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
+public class Changer<T, R> {
```

Comment:
Javadocs would drastically reduce the time spent on reading the code

Reply:
Done